packages/esm-patient-registration-app/src/widgets/display-photo.test.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import DisplayPatientPhoto from './display-photo.component';
+import { mockPatient } from '../../../../__mocks__/appointments.mock';
+
+jest.mock('../patient-registration/patient-registration.resource', () => ({
+  usePatientPhoto: jest.fn().mockReturnValue({ data: { imageSrc: 'test-image-src' } }),
+}));
+
+jest.mock('geopattern', () => ({
+  generate: jest.fn().mockReturnValue({
+    toDataUri: jest.fn().mockReturnValue('test-pattern-url'),
```

Comment:
Use a sample url instead of `test-pattern-url`

Reply:
done @Piumal1999